src/form/TextInput.tsx

```diff
@@ -60,19 +68,21 @@ export const TextInput: React.FC<MyProps> = (props) => {
           ref={myInput}
           name={name}
           id={name}
-          type="text"
           autoComplete="off"
-          className={`${formStyles.FormControl} ${triedToSubmit ? formStyles.FormSubmitted : ''}`}
+          className={`${formStyles.FormControl} ${triedToSubmit ? formStyles.FormSubmitted : ''} ${showUpDownArrows? "": formStyles.HideNumberUpDownArrows}`}
           placeholder={placeholder}
           onChange={valueChanged}
-          onFocus={onFocus}
-          onBlur={onBlur}
-          value={value || ""}
+          value={value || (value===0? value: "")}
           onKeyUp={handleEnter}
           readOnly={!!readOnly}
-          // disabled={!!readOnly}
+          disabled={!!readOnly}
```

Comment:
@Derryrover 
Hi Tom, I missed this one in the last review. Is there a reason for you to switch this disabled parameter on? The reason I commented it out is because I want the focus attribute to work even when the field is read-only (for the help text in the side column). I didn't document it in the code but it's necessary to do it this way. If it is not going to interfere with your work then I will remove them again. Is it ok?

Reply:
@hoanphungt
Sorry I already merged.
I will fix this now

Reply:
@Derryrover 
Ah don't worry about it! I fixed it in my PR: https://github.com/nens/lizard-management-client/pull/589